bpdm-common/src/main/kotlin/org/eclipse/tractusx/bpdm/common/service/BaseSyncRecordService.kt

```diff
@@ -92,14 +92,14 @@ abstract class BaseSyncRecordService<SYNC_TYPE : Enum<*>, SYNC_RECORD : BaseSync
     }
 
     @Transactional(isolation = Isolation.SERIALIZABLE)
-    open fun setSynchronizationSuccess(type: SYNC_TYPE): SYNC_RECORD {
+    open fun setSynchronizationSuccess(type: SYNC_TYPE, finishedAt: Instant?): SYNC_RECORD {
```

Comment:
This will lead to other overridden functions to fail. If you have a default null parameter, this is resolved